pyproject.toml

```diff
@@ -10,7 +10,7 @@ APScheduler = "3.10.4"
 cachetools = "^5.3.1"
 fastapi = "^0.88.0"
 framegrab = "^0.5.0"
-groundlight = "^0.17.6"
+groundlight = "<0.18.0"
```

Comment:
edge is not yet compatible w/ newest sdk

Reply:
Let's pin this to a specific version. We see this break on versions that should be compatible because the edge uses SDK internals directly.